internal/services/springcloud/spring_cloud_elastic_application_performance_monitoring_resource_test.go

```diff
@@ -0,0 +1,168 @@
+package springcloud_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/appplatform/2023-09-01-preview/appplatform"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type SpringCloudElasticApplicationPerformanceMonitoringResource struct{}
+
+func TestAccSpringCloudElasticApplicationPerformanceMonitoringPerformanceMonitoring_basic(t *testing.T) {
```

Comment:
All the test names have duplicate `PerformanceMonitoring` at the end
```suggestion
func TestAccSpringCloudElasticApplicationPerformanceMonitoring_basic(t *testing.T) {
```